src/test/java/org/hillel/homework/NumOfGoodPairsTest.java

```diff
@@ -0,0 +1,32 @@
+package org.hillel.homework;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+public class NumOfGoodPairsTest {
+    @Test
+    public void test1() {
+        assertEquals(4, NumOfGoodPairs.numIdenticalPairs(new int[]{1, 2, 3, 1, 1, 3}));
+    }
+
+    @Test
+    public void test2() {
+        assertEquals(0, NumOfGoodPairs.numIdenticalPairs(new int[]{}));
+    }
+
+    @Test
+    public void test3() {
+        assertEquals(0, NumOfGoodPairs.numIdenticalPairs(new int[]{1, 2, 3}));
+    }
+
+    @Test
+    public void test4() {
+        assertEquals(0, NumOfGoodPairs.numIdenticalPairs(new int[]{1}));
+    }
+
+    @Test
+    public void test5() {
```

Comment:
Give more clear names to the tests. Try to use give_when_then pattern:numIdenticalPairs_NullArra_NPEIsTrown, or something